custom/reports/mc/models.py

```diff
@@ -626,3 +628,9 @@ class Meta:
 
 
 MalariaConsortiumFluffPillow = MalariaConsortiumFluff.pillow()
+
+
+def get_pillow():
+    return MalariaConsortiumFluffPillow(
+        change_feed=KafkaChangeFeed(topic=topics.FORM, group_id='mc-fluff')
```

Comment:
it's the consumer group stuff from the presentation (lets you have multiple subscribers that get distributed messages from the same topic). we aren't using it for anything now